samples/bluetooth/hci_uart/boards/nrf54l15dk_nrf54l15_cpuapp.overlay

```diff
@@ -4,6 +4,9 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-&wdt31 {
+&uart20 {
```

Comment:
??

Reply:
Rename of the file does not change the file content.